contrib/endpoints/include/BUILD

```diff
@@ -41,6 +41,7 @@ filegroup(
     srcs = [
         "api_manager/api_manager.h",
         "api_manager/compute_platform.h",
+        "api_manager/config_manager.h",
```

Comment:
do we need to expose this interface?  I think this can be an internal to api_manager.